### PR TITLE
Android agent documents update

### PR DESCRIPTION
**Pull request by ywang-nr**
* What problems does this PR solve? - Update documentations to clarify API usage

### CODE REVIEW

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/set-max-event-buffer-time.mdx [hunk 1]
```diff
@@ -26,7 +26,13 @@ Agent version 5.0.0 or higher.
 
 ## Description
 
-This method sets the maximum time (in seconds) that the New Relic Android agent will store events in memory before sending the buffered events to New Relic. In other words, this method controls the length of the [harvest cycle](/docs/apm/new-relic-apm/getting-started/glossary#harvest-cycle).
+This method sets the maximum time in seconds that the New Relic Android agent will store events in memory before ncluding the buffered event to be sent to New Relic in the next [harvest cycle](/docs/apm/new-relic-apm/getting-started/glossary#harvest-cycle).
```
Comment (cthomas-newrelic) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/set-max-event-buffer-time.mdx:29]: typo: 'ncluding'

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/set-max-event-buffer-time.mdx [hunk 2]
```diff
@@ -26,7 +26,13 @@ Agent version 5.0.0 or higher.
...
+
+* Default is 600 seconds (10 minutes).
+* Minimum value cannot be less than 60 seconds.
+* Maximum value cannot be greater than 60 seconds.
```
Comment (cthomas-newrelic) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/set-max-event-buffer-time.mdx:33]: "maximum value _should not_ be greater than **600** seconds"

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/set-max-event-pool-size.mdx [hunk 3]
```diff
@@ -26,7 +26,11 @@ Agent version 5.0.0 or higher.
 
 ## Description
 
-By default, mobile monitoring collects a maximum of 1,000 events per [harvest cycle](/docs/apm/new-relic-apm/getting-started/glossary#harvest-cycle) (cycles are ten minutes long by default). This method lets you override the maximum size of that event pool. When the pool size limit is reached, the New Relic Android agent will begin [sampling events](/docs/agents/manage-apm-agents/agent-data/new-relic-events-limits-sampling), discarding some old and some new events, until the pool of events are transmitted with the next harvest cycle.
+By default, mobile monitoring collects a maximum of 1,000 events per [harvest cycle](/docs/apm/new-relic-apm/getting-started/glossary#harvest-cycle) (cycles are ten minutes long by default).
```
Comment (cthomas-newrelic) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/set-max-event-pool-size.mdx:29]: Per _event harvest_ cycle, which is 600 secs (default). The normal harvest cycle is 60 sec (fixed). Events are also harvested when the app goes to background.

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/set-max-event-pool-size.mdx [hunk 4]
```diff
@@ -26,7 +26,11 @@ Agent version 5.0.0 or higher.
...
+By default, mobile monitoring collects a maximum of 1,000 events per [harvest cycle](/docs/apm/new-relic-apm/getting-started/glossary#harvest-cycle) (cycles are ten minutes long by default). 
+This method controls the maximum size of the event pool stored in the memory until the next harvest cycle. When the pool size limit is reached, the New Relic Android agent will begin sampling events, discarding some old and some new events, until the pool of events are transmitted with the next harvest cycle. 
+This method lets you override the maximum size of that event pool. When the pool size limit is reached, the New Relic Android agent will begin [sampling events](/docs/agents/manage-apm-agents/agent-data/new-relic-events-limits-sampling), discarding some old and some new events, until the pool of events are transmitted with the next harvest cycle.
+
+* The default value for the event harvest cycle is 600 seconds.
```
Comment (cthomas-newrelic) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/set-max-event-pool-size.mdx:33]: It seems this should refer to the maxPoolSize value, not the time?

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/set-max-event-buffer-time.mdx [hunk 5]
```diff
@@ -29,7 +29,8 @@ Compatible with all agent versions.
 
 ## Description
 
-Sets the maximum time (in seconds) that the agent will store events in memory. When the oldest event exceeds this time, the agent transmits the buffered content at the [harvest cycle](/docs/accounts-partnerships/education/getting-started-new-relic/glossary#harvest-cycle). You must set this value after the start call.
+This method sets the maximum time in seconds that the New Relic iOS agent will store events in memory before including the buffered event to be sent to New Relic in the next [harvest cycle] (/docs/accounts-partnerships/education/getting-started-new-relic/glossary#harvest-cycle).
+In other words, when the oldest event timestamp exceeds this custom configured time, the agent transmits the buffered content at the harvest cycle between each connection from a New Relic agent to the collector. You must set this value after the start call.
```
Comment (cthomas-newrelic) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/set-max-event-buffer-time.mdx:33]: "_after the start call_" . Reads a little funny, perhaps change to "_after the "_Agent.start()_ call" (or insert iOS -specific method name)

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/set-max-event-pool-size.mdx [hunk 6]
```diff
@@ -28,10 +28,12 @@ Compatible with all agent versions.
...
-
+By default, the iOS agent collects a maximum of 1000 events per [harvest cycle](/docs/accounts-partnerships/education/getting-started-new-relic/glossary#harvest-cycle). 
+This method controls the maximum size of the event pool stored in the memory until the next harvest cycle. When the pool size limit is reached, the New Relic Android agent will begin [sampling events](/docs/agents/manage-apm-agents/agent-data/new-relic-events-limits-sampling), discarding some old and some new events, until the pool of events are transmitted with the next harvest cycle. 
 This method lets you override the maximum size of that event pool. You must set this value after the start call.
 
+* The default value for the event harvest cycle is 600 seconds.
```
Comment (cthomas-newrelic) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/set-max-event-pool-size.mdx:35]: Same as for Android: seems it should refer to the maxPoolSize() method instead?

**Review by cthomas-newrelic — CHANGES_REQUESTED** [review]
We're getting there...

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/set-max-event-pool-size.mdx (hunk 4 above)
Comment (ywang-nr) [reply src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/set-max-event-pool-size.mdx:33]: so for this one, we are in the setMaxEventPool file, talking about the event harvest cycle, which the client can refer to the setMaxEventBufferTime for more information. Does that make sense?

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/set-max-event-pool-size.mdx (hunk 6 above)
Comment (ywang-nr) [reply src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/set-max-event-pool-size.mdx:35]: see above

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/set-max-event-buffer-time.mdx (hunk 5 above)
Comment (ywang-nr) [reply src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/set-max-event-buffer-time.mdx:33]: Will do. Checking with Matt.

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/set-max-event-pool-size.mdx (hunk 4 above)
Comment (cthomas-newrelic) [reply src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/set-max-event-pool-size.mdx:33]: Ah, I see. Yes it does make more sense.

**Review by cthomas-newrelic — DISMISSED** [review]
Looks great!

**Pull request merged**